src/WebJobs.Extensions.DocumentDB/Bindings/DocumentDBScriptBindingProvider.cs

```diff
@@ -74,7 +75,14 @@ public override Type DefaultType
                 {
                     if (Context.Access == FileAccess.Read)
                     {
-                        return typeof(JObject);
+                        if (string.Compare("many", Context.Cardinality, StringComparison.OrdinalIgnoreCase) == 0)
+                        {
+                            return typeof(IEnumerable<JObject>);
```

Comment:
Feels stranger than JArray, but makes sense in the context of that email thread. Hopefully we can light up JArray out of the box. For DocDB converters you only need to support IEnumerable<T>, correct?